frontend/app/src/components/section-components/chat/create-channel-form.tsx

```diff
@@ -1,13 +1,10 @@
 import { Dispatch, useEffect, useState } from 'react';
```

Comment:
For slow connections, after creating a form the user has a 404 displayed, then the message box is loaded